internal/tools/aws/ec2.go

```diff
@@ -163,3 +167,29 @@ func GetSecurityGroupsWithFilters(filters []*ec2.Filter) ([]*ec2.SecurityGroup,
 
 	return sgOutput.SecurityGroups, nil
 }
+
+// IsValidAMI check if the provided AMI exists
+func (a *Client) IsValidAMI(AMIImage string) (bool, error) {
```

Comment:
What happens if the AMI string passed in is `""` which is the default if no AMI is provided? Does this work okay?

Reply:
humm, so we have "" if is default kops image? i thought it fulfill with the default kops image and never will be blank

so we need to check if is blank we bypass, sounds good? @gabrieljackson

Reply:
Yeah, exactly!

Here is an example of what the default kops metadata looks like:

`{"Name":"oifityg3r7b4z8cyfc9uo66gea-kops.k8s.local","Version":"latest","AMI":""}`

Reply:
got it